assets/src/edit-story/components/panels/design/captions/captions.js

```diff
@@ -49,7 +71,10 @@ export const MIN_MAX = {
 function CaptionsPanel({ selectedElements, pushUpdate }) {
   const tracks = getCommonValue(selectedElements, 'tracks', []);
   const isMixedValue = tracks === MULTIPLE_VALUE;
-  const captionText = __('Upload captions', 'web-stories');
+  const captionText = __('Upload a file', 'web-stories');
+  const clearFileText = __('Remove file', 'web-stories');
+  // don't have a way to tell if there was an error on upload yet
+  const uploadError = false;
```

Comment:
File upload still happens in the modal. We haven't switched away from that yet which means that we can't tell if a file had trouble being uploaded.

This will need to be added later, unless I am mistaken!